FAQ.md

```diff
@@ -29,4 +29,4 @@ added.
 
 ### How do I migrate from the GAIA/Kotlin version of AIF to the Java version?
 
-Refer to the [AIF v1.0.4 FAQ](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/blob/tag/v1.0.4/FAQ.md)
\ No newline at end of file
+Refer to the [AIF v1.0.5 FAQ](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/blob/tag/v1.0.5/FAQ.md)
```

Comment:
I'm not 100% sure, but I think this is supposed to point to develop, not 1.0.5.
[AIDA-1102](https://nextcentury.atlassian.net/browse/AIDA-1102) states:
Link to AIF should specify branch, e.g., AIDA-Interchange-Format/tree/develop

That said, AIDA-1102 mentions `README.md`, `java/README.md`, and `python/README.md`, but does not mention `FAQ.md`.

Of course, even if it links to develop, what should the link text say?  Maybe just `AIF FAQ` without a version number?  We may need @craigwarsaw to weigh in.

Reply:
Maybe the thinking is that since this is just a documentation link, it should just point to the current latest release of AIF, which is about to be 1.0.5.  If that's the thinking, I can buy that and will approve the PR.